### PR TITLE
Some fixes (but not: webServerPort not available, blank browser tab)

### PR DESCRIPTION
**Pull request by janilahti**
This PR addresses following issues:
* ~~#4 Show information message and don't launch browser if `webServerPort` is not available (reserved by another process).~~ Removed from this PR to be added later with enchancement.
* #8 Print PHP files correctly.
* #9 Add more paper sizes (a3, a5, legal).

### CODE REVIEW

**Comment by nobuhito** [comment]
@janilahti Thank you for having lots of patches. I am sorry that the reply was delayed. (March was busy at the end of the term)

I tried to check the patch you got from. However, we could not reproduce the error of 'EADDRINUSE' here.

Could you just change this part to another PR? I think that I want to release other parts immediately.

Thank you for your consideration.

**Comment by nobuhito** [comment]
@janilahti Looking at [here](https://nodejs.org/api/net.html#net_server_listen), it is likely to be possible to specify port usage errors with `e.code === "EADDRINUSE"`.

**Comment by janilahti** [comment]
Can you merge the just the specific commits from this PR if you want to leave out the #4 / be42d08 part for now? Github wants to accumulate my commits to one PR as long as it's open. I can modify and resubmit that one later if needed. The acute problem for most use users was mitigated already by changing the default port.

Plain `server.on('error' ...` worked for me on Linux and Windows platform, and I didn't see the need to check for specific errors -- although there could be a good case for more fine-grained checks: If the user specifies port number <1024 (and is not running vscode as root), binding to port naturally fails, and we might want to give a different error message.

File: src/extension.js [hunk 1]
```diff
@@ -16,37 +17,63 @@ function activate(context) {
...
+        server = null;
+      }
+
       if (server == null) {
         server = http.createServer(requestHandler);
+        server.on('error', function (err) {
+            vscode.window.showInformationMessage(
+                `Unable to print: Port ${port} is in use. \
+                Please set different port number in User Settings: printcode.webServerPort \
+                or stop the active server which has reserved the port.`
```
Comment (janilahti) [inline src/extension.js]: Could be better as:
`or stop the process that has reserved the port.`

**Comment by nobuhito** [comment]
@janilahti If you remove it, I merge it the next day. Let's discuss the message separately. 
(I am poor at English, so I will leave it to you😁)

The second comment thought that it was better to limit Error to "EADDRINUSE". It is not a retry.

**Comment by janilahti** [comment]
Commit you requested is now removed. I'll review it and make a new PR with enchanced functionality later.

**Comment by nobuhito** [comment]
@janilahti Thanks.

**Pull request merged**